ios/brave-ios/Sources/BraveWallet/Resources/zh.lproj/BraveWallet.strings

```diff
@@ -2149,12 +2107,6 @@
 /* The footer of the toggle for user to enable/disable NFT discovery inside Web3 settings. */
 "wallet.web3SettingsEnableNFTDiscoveryFooter" = "使用第三方 API 将您拥有的 NFT 自动新增至钱包。[了解更多](%@)";
 
-/* Description of the wrong IPFS gateway alert */
-"wallet.wrongGatewayAlertDescription" = "提供了无效的链接。网关应能够解析 IPFS 资源。";
-
-/* Title of the wrong IPFS gateway alert */
-"wallet.wrongGatewayAlertTitle" = "无法设置网关";
-
 /* The title of the sort option that sorts alphabetically from Z to A. Used in Portfolio/NFT filters and display settings. */
 "wallet.zToASortOption" = "Z 至 A";
 
```

Comment:
we dont need to remove/add strings in any xx.lproj/BraveWallet.strings files since they are generated files by script. 
as long as updates/changes are applied in WalletStringg.swift :)